projects/plugins/jetpack/_inc/client/newsletter/email-settings.jsx

```diff
@@ -122,6 +162,47 @@ const EmailSettings = props => {
 					onChange={ handleSubscriptionEmailsUseExcerptChange }
 				/>
 			</SettingsGroup>
+			{ isNewsletterFromNameEnabled && (
+				<SettingsGroup
+					hasChild
+					disableInOfflineMode
+					disableInSiteConnectionMode
+					module={ subscriptionsModule }
+					className="newsletter-group"
+				>
+					<FormLegend className="jp-form-label-wide">{ __( 'Sender name', 'jetpack' ) }</FormLegend>
+					<p>
+						{ __(
+							"This is the name that appears in subscribers' inboxes. It's usually the name of your newsletter or the author.",
+							'jetpack'
+						) }
+					</p>
+					<Container horizontalGap={ 0 } fluid className="sender-name">
+						<Col sm={ 3 } md={ 4 } lg={ 4 }>
+							<TextInput
+								value={ fromNameState.value }
+								disabled={ fromNameInputDisabled }
+								onChange={ handleSubscriptionFromNameChange }
+								placeholder={ siteName || __( 'Enter sender name', 'jetpack' ) }
+							/>
+						</Col>
+						<Col sm={ 1 } md={ 1 } lg={ 1 }>
+							<Button
+								primary
+								rna
+								onClick={ handleSubscriptionFromNameChangeClick }
+								disabled={ fromNameInputDisabled || ! fromNameState.hasChanged }
+							>
+								{ __( 'Save', 'jetpack' ) }
+							</Button>
+						</Col>
+						<Col className="sender-name-example">
+							{ __( 'Example: ', 'jetpack' ) } { fromNameState.value || siteName }{ ' ' }
```

Comment:
Let's use placeholder in "Example: %s" to make it easier to translate. Some translator will for sure leave the space out for example, and RTL languages will work better.

Reply:
Good call!